src/grafana/chart/templates/datasources.yaml

```
@@ -30,3 +30,6 @@ data:
         implementation: prometheus
         handleGrafanaManagedAlerts: true
       url: http://kube-prometheus-stack-alertmanager.monitoring.svc.cluster.local:9093
+{{- if .Values.extraDatasources }}
+  {{ toYaml .Values.extraDatasources | indent 4 }}
+{{- end }}
```

```suggestion
    {{- if .Values.extraDatasources }}
    {{- toYaml .Values.extraDatasources | nindent 4 }}
    {{- end }}
```

Part of this is preference (align templating for more readability), but currently this templates out with incorrect spacing because of the leading 2 spaces + 4 indented spaces. Using `-` to trim leading space + `nindent` will ensure this aligns right.